package.json

```diff
@@ -83,8 +85,6 @@
     "@date-io/date-fns": "^1.3.13",
     "@material-ui/pickers": "^3.3.10",
     "@nosferatu500/react-sortable-tree": "^3.0.5",
-    "ag-grid-community": "^26.1.0",
-    "ag-grid-react": "^26.1.0",
```

Comment:
Moving agGrid from dependencies to peerDependencies may break the documentation generation of the Table component.

Furthermore, the new Table prop 'agTheme' is only a string, I wonder if we really need to move the ag-grid-community dependency from the ui repo to the webapp: it could probably be kept in the ui repo.